looper/models.py

```diff
@@ -934,11 +934,27 @@ def add_sample_sheet(self, csv=None):
                             # so we don't re-derive them later.
                             merged_cols = {
                                     key: "" for key in merge_rows.columns}
-                            for row in merge_rows.index:
+                            for row in range(len(merge_rows.index)):
```

Comment:
This was bothering me, too; it feels dirty (at least in `pandas` to use the integer indexing). I stumbled across `iterrows` in the `pandas` docs, which I don't remember using before, but I've put that in here and in one other spot since it sounds built for this purpose and feels most straightforward with what the iteration is doing.